open-banking-accelerator/components/com.wso2.openbanking.accelerator.gateway/src/main/java/com/wso2/openbanking/accelerator/gateway/executor/dcr/DCRExecutor.java

```diff
@@ -610,6 +250,409 @@ public void postProcessRequest(OBAPIRequestContext obapiRequestContext) {
         }
     }
 
+    private void processPostRequestResponse(OBAPIResponseContext obapiResponseContext, String basicAuthHeader,
```

Comment:
Add method comment

Reply:
Resolved in [8657c32](https://github.com/wso2/financial-open-banking/pull/62/commits/8657c32a1adada4bcf5b652287b825b09a36ca96)